package-lock.json

```diff
@@ -16,12 +16,15 @@
         "concurrently": "^7.2.1",
         "cors": "^2.8.5",
         "express": "^4.18.1",
+        "google-auth-library": "^8.0.3",
```

Comment:
is this file generated? if it is, you should add this to your .gitignore so the changes aren't tracked.